service-front/public/assets/v2/js/application.js

```diff
@@ -23041,6 +23041,190 @@ var SessionTimeoutDialog = function (options) {
+                            document.cookie = cookie + '=;expires=' + new Date() + ';domain=.' + window.location.hostname + ';path=/'
+                        }
+                    }
+                }
+            }
+        }
+
+        GOVUK.setCookie('cookie_policy', JSON.stringify(cookieConsent), { days: 365 })
+    }
+
+    GOVUK.checkConsentCookieCategory = function (cookieName, cookieCategory) {
+        var currentConsentCookie = GOVUK.getConsentCookie()
+
+        // If the consent cookie doesn't exist, but the cookie is in our known list, return true
+        if (currentConsentCookie === null) {
+            return false
+        }
+
+        currentConsentCookie = GOVUK.getConsentCookie()
+
+        // Sometimes currentConsentCookie is malformed in some of the tests, so we need to handle these
+        try {
+            return currentConsentCookie[cookieCategory]
+        } catch (e) {
+            console.error(e + ' when checking ' + cookieName + ' and ' + cookieCategory)
+            return false
+        }
+    }
+
+    GOVUK.checkConsentCookie = function (cookieName, cookieValue) {
+        // If we're setting the consent cookie OR deleting a cookie, allow by default
+        if (cookieName === 'cookie_policy' || (cookieValue === null || cookieValue === false)) {
+            return true
+        }
+
+        if (COOKIE_CATEGORIES[cookieName]) {
+            var cookieCategory = COOKIE_CATEGORIES[cookieName]
+
+            return GOVUK.checkConsentCookieCategory(cookieName, cookieCategory)
+        } else {
+            // Deny the cookie if it is not known to us
+            return false
+        }
+    }
+
+    GOVUK.setCookie = function (name, value, options) {
+        if (GOVUK.checkConsentCookie(name, value)) {
+            if (typeof options === 'undefined') {
+                options = {}
+            }
+            var cookieString = name + '=' + value + '; path=/'
+            if (options.days) {
+                var date = new Date()
+                date.setTime(date.getTime() + (options.days * 24 * 60 * 60 * 1000))
+                cookieString = cookieString + '; expires=' + date.toGMTString()
+            }
+            if (document.location.protocol === 'https:') {
+                cookieString = cookieString + '; Secure'
+            }
+            document.cookie = cookieString
+        }
+    }
+
+    GOVUK.getCookie = function (name) {
+        var nameEQ = name + '='
+        var cookies = document.cookie.split(';')
+        for (var i = 0, len = cookies.length; i < len; i++) {
+            var cookie = cookies[i]
+            while (cookie.charAt(0) === ' ') {
+                cookie = cookie.substring(1, cookie.length)
+            }
+            if (cookie.indexOf(nameEQ) === 0) {
+                return decodeURIComponent(cookie.substring(nameEQ.length))
+            }
+        }
+        return null
+    }
+
+    global.GOVUK = GOVUK
+}(window))
+;
 this["lpa"] = this["lpa"] || {};
```

Comment:
What's the purpose of these 2 lines here to do with an LPA?

Reply:
This is a compiled file and can be ignored for the purposes of PRs

Thats said I believe these two lines are specific to the javascript templating components that are built and show here due to the way diffing works.